bundle/tests/sync_include_exclude_no_matches_test.go

```diff
@@ -42,3 +42,11 @@ func TestSyncIncludeExcludeNoMatchesTest(t *testing.T) {
 	require.Equal(t, diags[2].Severity, diag.Warning)
 	require.Contains(t, summaries, diags[2].Summary)
 }
+
+func TestSyncIncludeWithNegate(t *testing.T) {
```

Comment:
Can you also add a positive test for this, ie what the diagnostic looks like with a negative glob pattern?